src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write code here
+document.addEventListener('DOMContentLoaded', () => {
+  const tree = document.querySelector('.tree');
```

Comment:
Ensure that the element with the class 'tree' exists in the DOM. If it doesn't, `document.querySelector('.tree')` will return `null`, and subsequent operations will fail.